source/index.html.md

```diff
@@ -630,6 +630,27 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects.
+Currently only the JWE is accepted.
+
+<dl class="dl-vertical">
+  <dt>samsungpay[jwt]</dt>
+  <dd>
+    <a href="https://tools.ietf.org/html/rfc7516#section-3.1"><span class="type">Compact serialized JWE token</span></a>
```

Comment:
I am no HTML expert, but wouldn't it normally be `<span><a>text</a></span>`?

Reply:
For `span` it doesn't matter. 

But I've changed it to `div` in my local code, it matters there. So I've swapped places for them.